steam-login/usr/bin/steam-de

```diff
@@ -94,7 +104,3 @@ while check_steam_window
 do
     sleep 10
 done
-
-#Exit steam nicely...
-steam -shutdown
```

Comment:
Removing this may be an issue leaving a steam program running in the background. If the -shutdown doesn't works any more we should find a replacement, even it's a system "kill"

Reply:
Okay,
In theory that possible to run steam by (i didn't test that for now):
systemd-run --user -E DISPLAY=..., if that acceptable of course, and on exit do something like: systemctl --user stop run-...